CHANGES.rst

```diff
@@ -4,7 +4,33 @@ Changelog for zest.releaser
 5.8 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Always exit with error code 1 when we exit explicitly.  In some
+  cases we would exit with success code 0 when we exited based on the
+  answer to a question.  This happened when the user did not want us
+  to create the missing ``tags`` directory in subversion, and also
+  after asking if the user wanted to continue even though 'nothing
+  changed yet' was in the history.
+  [maurits]
+
+- Added ``required_changelog_text`` to the prerelease data.  This is
+  text that must be present in the changelog.  It can be a string or a
+  list, for example ``["New:", "Fixes:"]``.  For a list, only one of
+  them needs to be present.
+  [maurits]
+
+- Look for the 'Nothing changed yet' text in the complete text of the
+  history entries of the current release, instead of looking at it
+  line by line.  This means that third party code can overwrite
+  ``nothing_changed_yet`` in the prerelease data to span multiple lines.
+  [maurits]
```

Comment:
I had to read this one a couple of times before I understood what was happening. If you say "zest.releaser extensions" instead of "third party code", it already helps.

Similarly "extensions can now tell zest.releaser to look for specific required words in the changelog" is clearer than "Added ... to the prerelease data". Even I have to read it twice :-)